Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM composer AS composer
```

Comment:
I think the docker files should be moved to a `contrib` folder to clean up the root folder?